libass/ass_shaper.c

```diff
@@ -466,10 +466,6 @@ bool ass_create_hb_font(ASS_Font *font, int index)
 static hb_font_t *get_hb_font(ASS_Shaper *shaper, GlyphInfo *info)
 {
     ASS_Font *font = info->font;
-    hb_font_t *hb_font = font->hb_fonts[info->face_index];
```

Comment:
docs a few line above need to be updated; it no longer returns a cache-owned font instance